db/migrate/20160118115624_add_estimation_time_to_read.rb

```diff
@@ -0,0 +1,5 @@
+class AddEstimationTimeToRead < ActiveRecord::Migration
+  def change
+    add_column :links, :time_to_read, :string
```

Comment:
in this case, for example if we post a link to a book, we will just skip writing this estimation